### ISSUE TITLE
Clone Stamp cursor incorrect when zoomed at non-integer value

### ISSUE THREAD

**Issue by Ajcm20** [photopea/photopea#2619]
Hi,

When I use the clone tool is shown in red and it won't let me see the part of the photo I select. So when he took it to the part that I need it doesn't fit.

I will stay tuned.

Thanks
![Nuevo proyecto](https://user-images.githubusercontent.com/74681497/99581622-0849c500-29af-11eb-98f0-433225c0275a.jpg)

**Comment by photopea** [comment]
Hi, you probably tried to hold Alt and click on a Chromebook, am I right?

Alt+click is always converted to a right-click on Chromebooks. You need to disable this feature, as described here:

https://support.google.com/chromebook/thread/14888778?hl=en

**Comment by Ajcm20** [comment]
No actually what I do is press K + click for to activate the clone tool. But, I try to do what you say and it doesn´t work

**Comment by photopea** [comment]
Are you sure you never pressed Alt and clicked?

**Comment by Ajcm20** [comment]
Yes I never used that option

**Comment by photopea** [comment]
Can you help me reproduce the problem? After I open www.Photopea.com , what should I do to make the red circle appear?

**Comment by Ajcm20** [comment]
I open the image that i would edit, then i select the clonne tool press K + Click

**Comment by photopea** [comment]
@avagp can you reproduce it?

**Comment by photopea** [comment]
@Ajcm20 I tried to do it in Chrome in Windows and it works well, I can not see any red circle.

Are you using a Chromebook? Are you sure you can reproduce it at least twice?

**Comment by Ajcm20** [comment]
No, I also tried to use photophea in another computer and it appear the same red circle

**Comment by Ajcm20** [comment]
It appear since the last sunday, before that everything was  fine

**Comment by photopea** [comment]
@Ajcm20 So what operating system and a web browser are you using?

**Comment by Ajcm20** [comment]
My operating system is intel corel i5 and I´m using Chrome

**Comment by photopea** [comment]
Intel is not an operating system. Can you take a picture of the screen of your computer, when it is on, and share that picture with us?

**Comment by Ajcm20** [comment]
Sorry I think is windows 8, but nevertheless I send you the photo![Nuevo proyecto](https://user-images.githubusercontent.com/74681497/99595784-48b33e00-29c3-11eb-84a2-c34651f64e03.jpg)

**Comment by photopea** [comment]
Ok, thanks! BTW. do you do a left mouse click or a right mouse click in Photopea? You should do a left mouse click to choose a clone source.

**Comment by Ajcm20** [comment]
I do left mouse click

**Comment by Ajcm20** [comment]
yes it have the same problem

**Comment by photopea** [comment]
What version of Chrome are you using? Can you update it?

**Comment by Ajcm20** [comment]
I have this Versión 86.0.4240.198 (Build oficial) (64 bits). Umm let me try

**Comment by photopea** [comment]
Could you record a video of opening Photopea and cloning, so that I could see what you are actually doing?

The red circle appears when you hold Alt and press the right mouse button. But if you never pressed Alt or Right-click, it should never appear.

**Comment by Ajcm20** [comment]
I updated to Versión 87.0.4280.66 (Build oficial) (64 bits) an its the same

**Comment by photopea** [comment]
Could you record a video of opening Photopea and cloning, so that I could see what you are actually doing?

The red circle appears when you hold Alt and press the right mouse button. But if you never pressed Alt or Right-click, it should never appear.

**Comment by Ajcm20** [comment]
I cannot upload here. Where can I send you?

**Comment by Ajcm20** [comment]
the video

**Comment by photopea** [comment]
you can send it to support@photopea.com , or upload to youtube and share a link.

**Comment by Ajcm20** [comment]
Thanks, I already send you the video

**Comment by photopea** [comment]
Ok, this is a really a bug :D you forgot to tell me that you "zoomed in with dragging". It does not happen if you zoom only with clicking or with Ctrl + - .

I will try to fix it tomorrow.

**Comment by Ajcm20** [comment]
Oh Im so sorry for not telling you, Im totally forgot that. But you right!!! it works!!! Thank you so much,

**Comment by avagp** [comment]
Hi!

I had to add something to the report.

The bug in 2296 was "fixed". I mean in "", because, it returned to a past stage, where i press Alt+Left Click and i can change the size of the brush.

I have returned to the default chrome os option: Alt + Left Click = Right Click

So... maybe it has something to do with it.

Even though the problem Ajcm20 is indicating is the same bug i had on 2296.

I am attaching a video:
https://video.drift.com/v/abUFhmJvvfh/
